viadot/sources/genesys.py

```diff
@@ -323,21 +315,17 @@ def download_all_reporting_exports(
                 )
                 continue
 
-            if single_report[4].lower() == "queue_performance_detail_view":
-                file_name = (
-                    temp_ids_mapping.get(single_report[2]) + "_" + single_report[3]
-                ).upper()
+            date = self.start_date.replace("-", "")
+            if single_report[4].lower() in [
+                "queue_performance_detail_view",
+                "agent_status_detail_view",
+            ]:
+                file_name = self.view_type.upper() + f"_{next(self.count)}_" + f"{date}"
```

Comment:
You can use also fstring in this way: `f"{self.view_type.upper()}_{next(self.count)}_{date}"`

Reply:
You are right, this is a legacy from the first version. Changed